pom.xml

```diff
@@ -665,74 +665,74 @@
         <apache.tomcat.dependency.version>10.1.4</apache.tomcat.dependency.version>
         <java.websocket.version>1.5.3</java.websocket.version>
-        <slf4j.api.version>1.7.30</slf4j.api.version>
-        <json.smart.version>2.4.7</json.smart.version>
+        <slf4j.api.version>2.0.6</slf4j.api.version>
+        <json.smart.version>2.4.8</json.smart.version>
         <junit.version>4.13.2</junit.version>
         <log4j.version>2.19.0</log4j.version>
         <maven.checkstyle.plugin.version>3.0.0</maven.checkstyle.plugin.version>
         <maven.checkstyle.version>8.18</maven.checkstyle.version>
         <maven.spotbugs.version>4.0.4</maven.spotbugs.version>
         <maven.surefire.plugin.version>2.22.2</maven.surefire.plugin.version>
-        <nimbus.jose.jwt.version>7.9</nimbus.jose.jwt.version>
+        <nimbus.jose.jwt.version>8.0</nimbus.jose.jwt.version>
         <org.json.version>3.0.0.wso2v1</org.json.version>
         <os.mvn.plugin.version>1.6.2</os.mvn.plugin.version>
-        <otel.alpha.version>1.18.0-alpha</otel.alpha.version>
-        <otel.version>1.18.0</otel.version>
-        <otel.otlp.version>1.18.0</otel.otlp.version>
-        <shade.plugin.version>3.2.4</shade.plugin.version>
-        <slf4j.version>1.7.21</slf4j.version>
-        <swagger.v3.models.version>2.0.8</swagger.v3.models.version>
-        <testcontainers.version>1.17.5</testcontainers.version>
+        <otel.alpha.version>1.22.0-alpha</otel.alpha.version>
+        <otel.version>1.22.0</otel.version>
+        <otel.otlp.version>1.22.0</otel.otlp.version>
+        <shade.plugin.version>3.4.1</shade.plugin.version>
+        <slf4j.version>2.0.6</slf4j.version>
+        <swagger.v3.models.version>2.2.8</swagger.v3.models.version>
+        <testcontainers.version>1.17.6</testcontainers.version>
         <testng.version>6.11</testng.version>
         <tomcat.annotations.api.version>6.0.53</tomcat.annotations.api.version>
         <toml4j.version>0.7.2</toml4j.version>
         <analytics.common.version>6.1.63</analytics.common.version>
         <apache.commons.version>1.5.6.wso2v1</apache.commons.version>
         <lmax.version>3.4.2.wso2v1</lmax.version>
-        <download.maven.plugin.version>1.6.1</download.maven.plugin.version>
+        <download.maven.plugin.version>1.6.8</download.maven.plugin.version>
         <apictl.version>4.2.0-alpha</apictl.version>
         <apim.version>4.2.0-alpha</apim.version>
         <grpc.healthProbe.version>0.4.15</grpc.healthProbe.version>
         <analytics.publisher.client.version>1.1.0</analytics.publisher.client.version>
-        <awaitility.version>3.1.2</awaitility.version>
-        <apache.httpcomponents.version>4.5.13</apache.httpcomponents.version>
+        <awaitility.version>4.2.0</awaitility.version>
+        <apache.httpcomponents.version>4.5.14</apache.httpcomponents.version>
         <jackson.databind.version>2.14.1</jackson.databind.version>
         <snakeyaml.version>1.33</snakeyaml.version>
```

Comment:
can't we remove snakeyaml dependency?

Reply:
Sure. I'll check it and update here.

Reply:
We can remove snakeyaml dependency as well.